CONTRIBUTING.md

```diff
@@ -5,11 +5,11 @@ If you would like to contribute to the curriculum, please [sign up here](https:/
 **Get started:**
 - Choose an issue from the [TO-DO](https://github.com/Techtonica/curriculum/projects/2) column that seems right for you & message Techtonica staff. If you know someone connected to Techtonica, they may be able to add you to our slack channel. If you are here for GSSoC19, message Alina L. (M) and say you'd like to pick up the issue.
```

Comment:
@vegetabill Don't you think GSSoC19 could be changed to GSSoC20 ?

Reply:
@Kundan28 there is a whole section above so we should actually remove that sentence. 

@habeeba-naaz please remove the "If you are here..." sentence please.

Reply:
I changed